src/screens/HomeScreen/component.js

```diff
@@ -124,12 +129,23 @@ const styles = StyleSheet.create({
     flex: 1,
     flexDirection: "row",
     flexWrap: "wrap",
+    justifyContent: "center",
     marginHorizontal: 12
   },
   tile: {
-    width: "50%",
-    paddingHorizontal: 4,
-    marginBottom: 8
+    width: "47%",
```

Comment:
This width shouldn't be necessary. I think it would be great if we could follow the same approach as on the other pages.

1. Use `ContentPadding`. 👉 This takes care of 8px padding on small devices, 16px on medium and no padding on landscape. We can then add `maxWidth: 440` to make it look good on landscape.
2. Make the 2 columns work the same way as in `SponsorLogoContainer`. Essentially adding padding only to every second item in the list.